src/blocks/stars.css

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+}
+
+.stars__sizing {
+  width: 16px;
+  height: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__sizing:not(:last-child) {
+  margin-right: 4px;
```

Comment:
Consider using a variable for spacing values like margin-right to maintain consistency and ease of maintenance. For example, `--space-small: 4px;` and then `margin-right: var(--space-small);`.